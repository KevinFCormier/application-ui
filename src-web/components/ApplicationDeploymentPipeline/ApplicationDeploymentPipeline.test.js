/*******************************************************************************
 * Licensed Materials - Property of IBM
 * (c) Copyright IBM Corporation 2016, 2019. All Rights Reserved.
 *
 * Note to U.S. Government Users Restricted Rights:
 * Use, duplication or disclosure restricted by GSA ADP Schedule
 * Contract with IBM Corp.
 *******************************************************************************/

import { BrowserRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../../../src-web/index';

const React = require('react');
const renderer = require('react-test-renderer');
const ApplicationDeploymentPipeline = require('../ApplicationDeploymentPipeline')
  .default;

describe('ApplicationDeploymentPipeline', () => {
  it('ApplicationDeploymentPipeline renders correctly.', () => {
    const tree = renderer.create(<Provider store={store}><Router><ApplicationDeploymentPipeline /></Router></Provider>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
