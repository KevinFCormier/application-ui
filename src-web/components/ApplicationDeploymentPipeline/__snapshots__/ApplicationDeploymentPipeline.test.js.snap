// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ApplicationDeploymentPipeline ApplicationDeploymentPipeline renders correctly. 1`] = `
<div
  id="DeploymentPipeline"
>
  Deployment Pipeline
</div>
`;
